common/annotations/my.xml

```diff
@@ -1381,7 +1381,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🐡" type="tts">ငါးပူဖောင်း</annotation>
 		<annotation cp="🦈">ငါး | ငါးမန်း</annotation>
 		<annotation cp="🦈" type="tts">ငါးမန်း</annotation>
-		<annotation cp="🐙">ဘဝှဲ ရဘဝှေဲ ဘဝှဲ ဘဝွဲ ဘဝြဲ ဘဝဲ</annotation>
+		<annotation cp="🐙">ဘဝှဲ ရဘဝှေဲ ဘဝှဲ ဘဝှဲ ဘဝွဲ ဘဝဲ</annotation>
```

Comment:
I think you have to revert this change in my.xml. (There was an earlier problem with the Zwagyi "fixer" in DAIP)

Reply:
fixed in c7b1ea206bbfc2ef399019aafd4350858803a7cc